src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +137,46 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
+  //get the info object from the server 
+  public ServerInfo getVersion() {
+
```

Comment:
I think I tried to follow the spacing with the above methods? (line 119) Is the double spacing the space after "public ServerInfo getVersion() {" (line 142) and after the try statement (144)? I just wanted to clarify on the double-spacing, thanks!